src/knx/group_object.cpp

```diff
@@ -279,3 +279,30 @@ void GroupObject::valueNoSend(const KNXValue& value, const Dpt& type)
 
     KNX_Encode_Value(value, _data, _dataLength, type);
 }
+
+bool GroupObject::valueModifiedSend(const KNXValue& value, const Dpt& type, const bool forceSend /*= false*/)
+{
+    if (_commFlag == Uninitialized)
```

Comment:
add forceSend here:
`if (forceSend || _commFlag == Uninitialized)`

Reply:
Would change (intended) behaviour: The return value should always indicate a changed value/state.

Starting from `Uninitialized` or setting the first value (assumption: no defined value and response to read requests) there is always a change. For following updates it depends on existing and new value.